pacaur

```diff
@@ -1103,9 +1103,70 @@ SearchInfoAur() {
     if [[ -z "$(grep -E "\-\-[r]?sort" <<< ${coweropts[@]})" ]]; then
         [[ $sortorder = descending ]] && coweropts+=("--rsort=$sortby") || coweropts+=("--sort=$sortby");
     fi
-    cower ${coweropts[@]} -- $@
-}
+    
+    # store cower information into an array, one line per package and values 
+    # separated by vertical slashes
+    formatopts='--listdelim # --format %n|%v|%d|%u|%p|%L|%P|%D|%M|%O|%C|%R|%m|%o|%r|%t|%s|%a\n'
+    readarray cower_array < <(cower ${formatopts} ${coweropts[@]} -- $@)
+
+    format_strings=( $"Name" $"Version" $"Description" $"URL" $"AUR Page" $"Licenses" $"Provides" $"Depends on" $"Make Deps" $"Optional Deps" $"Conflicts With" $"Replaces" $"Mantainer" $"Votes" $"Popularity" $"Out of Date" $"Submitted" $"Last Modified" ) 
+    
+    # in some languages more tabs are needed to properly adjust the info
+    format_tab=16
```

Comment:
Summary (taken from `pacman` format):

* **en_US.UTF-8 :** 16 
* __es_ES.UTF-8 :__ 26
* __it_IT.UTF-8 :__ 26
* __pl_PL.UTF-8 :__ 22
* __fr_FR.UTF-8 :__ 25
* __pt_PT.UTF-8 :__ 32

I can't see any rule checking the output. I think the most logical approach is to read LANG environmental variable and manually set `format_tab` for all the languages supported by `pacaur`.